crates/typos-cli/src/file.rs

```diff
@@ -664,6 +773,56 @@ fn fix_buffer(mut buffer: Vec<u8>, typos: impl Iterator<Item = typos::Typo<'stat
     buffer
 }
 
+fn fix_file_name<'a>(
+    path: &std::path::Path,
+    file_name: &'a str,
+    fixes: Vec<typos::Typo<'static>>,
+) -> Result<(), std::io::Error> {
+    let file_name = file_name.to_owned().into_bytes();
+    let new_name = fix_buffer(file_name, fixes.into_iter());
+    let new_name = String::from_utf8(new_name).expect("corrections are valid utf-8");
+    let new_path = path.with_file_name(new_name);
+    std::fs::rename(path, new_path)?;
+    Ok(())
+}
+
+fn select_fix(typo: &typos::Typo<'_>) -> Option<usize> {
+    if is_fixable(&typo) {
+        let confirmation = Confirm::new()
+            .with_prompt("Do you want to apply the fix suggested above?")
+            .default(true)
+            .show_default(true)
+            .interact()
+            .unwrap();
+
+        if confirmation {
+            return Some(0);
+        }
+    } else {
+        let mut items = match &typo.corrections {
+            typos::Status::Corrections(c) => c,
+            _ => return None,
+        }
+        .clone();
+        items.insert(0, std::borrow::Cow::from("None (skip)"));
+
+        let selection = Select::new()
+            .with_prompt("Please choose one of the following suggestions")
+            .items(&items)
+            .default(0)
+            .interact()
+            .unwrap();
+
+        if selection == 0 {
+            return None;
+        }
+
+        return Some(selection - 1);
```

Comment:
```suggestion
        .clone();

        items.insert(0, std::borrow::Cow::from("None (skip)"));
        let selection = Select::new()
            .with_prompt("Please choose one of the following suggestions")
            .items(&items)
            .default(0)
            .interact()
            .unwrap();
        if selection == 0 {
            return None;
        }
        return Some(selection - 1);
```

By grouping using blank lines, it helps clarify that these are all coupled together.  For example, I originally missed `items.insert` and was assuming `selection - 1` was a quirk of that API.